org.kde.dragonplayer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "app-id": "org.kde.dragonplayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-23.08",
+  "sdk": "org.kde.Sdk",
+  "command": "dragon",
+  "rename-icon": "dragonplayer",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--filesystem=home:ro",
```

Comment:
doesn't it use portals?

Reply:
I'm not too knowledgeable on portals. Are they preferred over this type of permissions?

Reply:
Yea

Reply:
https://github.com/KDE/dragon/blob/0e14b40c0407767d4217632c24fecfd00ca046c9/src/app/mainWindow.cpp#L657

Reply:
So if it uses portals to access things, what should I remove?

Reply:
The filesystem permission

Reply:
Removed.